### PR TITLE
docs(guides): clarified the config element in templated stage

### PR DESCRIPTION
**Pull request by dorbin**

### CODE REVIEW

**Comment by dorbin** [comment]
#1765

File: guides/user/pipeline/pipeline-templates/instantiate.md [hunk 1]
```diff
@@ -139,6 +139,12 @@ stage.
 ]
 ```
 
+### Set stage config-specific overrides
```
Comment (dorbin) [inline guides/user/pipeline/pipeline-templates/instantiate.md]:
```suggestion
The `config` object contains the entire stage config. In this Wait-stage example, `config` includes only `waitTime`. 
```

File: guides/user/pipeline/pipeline-templates/instantiate.md [hunk 2]
```diff
@@ -139,6 +139,12 @@ stage.
...
+
+You can set a value for any property in a stage, and that value is set in the stage when the pipeline is instantiated.
```
Comment (dorbin) [inline guides/user/pipeline/pipeline-templates/instantiate.md]:
```suggestion
```

File: guides/user/pipeline/pipeline-templates/instantiate.md [hunk 3]
```diff
@@ -139,6 +139,12 @@ stage.
...
+
+The example above shows a value for `waitTime` in the `config` element. When the pipeline is instantiated from this template, the `waitTime` is set to `67`.
```
Comment (dorbin) [inline guides/user/pipeline/pipeline-templates/instantiate.md]:
```suggestion
```

**Review by maggieneterval — APPROVED** [review]

**Pull request merged**